src/main/java/strategy/OperationTypes.java

```diff
@@ -0,0 +1,27 @@
+package strategy;
+
+import java.util.Map;
+import reporter.FruitShopDataReporter;
+
+public class OperationTypes {
+    private static final Map<String, OperationStrategy> operationsMap = Map.of(
+            "b", new AdditionalOperationStrategy(
+                    new FruitShopDataReporter().getCurrentBalanceService()),
+            "s", new AdditionalOperationStrategy(
+                    new FruitShopDataReporter().getCurrentBalanceService()),
+            "p", new PurchaseOperationStrategy(
+                    new FruitShopDataReporter().getCurrentBalanceService()),
+            "r", new AdditionalOperationStrategy(
+                    new FruitShopDataReporter().getCurrentBalanceService()));
+
+    public static OperationStrategy getOperationHandler(String operation) {
```

Comment:
don't use static methods please
and rename this class to OperationStrategy as I suggested earlier